### DISCUSSION TITLE
请问alist的用户有没上限？

### DISCUSSION THREAD

**Discussion by yuanhua123** [AlistGo/alist#6629] [Q&A] [answered]
自己搭建的alist在后台可以添加多少个用户？是否能看到每个用户的登录日志？

**Comment by pongfcnkl** [comment, marked as answer]
人多建议换mysql
启动时加--debug参数 从日志里就能看到每个用户做了啥

**Reply by yuanhua123** [reply]
请问如何增加启动时加--debug参数 这个功能呢

**Reply by pongfcnkl** [reply]
> 请问如何增加启动时加--debug参数 这个功能呢

alist server --debug
具体情况具体分析
